nbgrader/server_extensions/formgrader/formgrader.py

```diff
@@ -33,14 +33,11 @@ def init_tornado_settings(self, webapp):
 
         # Configure the formgrader settings
         tornado_settings = dict(
-            nbgrader_notebook_dir=self.coursedir.root,
-            nbgrader_notebook_dir_format=self.coursedir.directory_structure,
-            nbgrader_step=self.coursedir.autograded_directory,
+            nbgrader_coursedir=self.coursedir,
```

Comment:
It is needed, but it is accessed through the coursedir object rather than the notebook_dir variable.